src/Stripe.net/Infrastructure/Public/RequestTelemetry.cs

```diff
@@ -47,7 +47,8 @@ public void MaybeAddTelemetryHeader(IDictionary<string, string> headers)
         /// </summary>
         /// <param name="response">The HTTP response message.</param>
         /// <param name="duration">The request duration.</param>
-        public void MaybeEnqueueMetrics(HttpResponseMessage response, TimeSpan duration)
+        /// <param name="usage">Tracked behaviors.</param>
+        public void MaybeEnqueueMetrics(HttpResponseMessage response, TimeSpan duration, List<string> usage = null)
```

Comment:
I think this is technically breaking because anybody who subclassed RequestTelemetry would now have to implement a different signature, but I think this is unlikely enough that we can disregard.

Reply:
should we add an overload instead?

Reply:
Done!